plugins/logitech-hidpp/fu-logitech-hidpp-device.c

```diff
@@ -416,6 +417,13 @@ fu_logitech_hidpp_device_fetch_firmware_info(FuLogitechHidppDevice *self, GError
 			g_prefix_error(error, "failed to get firmware info: ");
 			return FALSE;
 		}
+		/* use the single available slot, otherwise -- the slot which is not active */
+		if (msg->data[0] == 0) {
+			if (!app_ok || FU_BIT_IS_CLEAR(msg->data[8], 0)) {
```

Comment:
no braces here for a single line -- then LGTM, thanks.

Reply:
Hm... @hughsie as wish. 
I thought it was better to use braces here for better readability, but doesn't matter.